src/MusicalScore/VoiceData/ChordSymbolContainer.ts

```diff
@@ -88,6 +154,7 @@ export class ChordSymbolContainer {
             text += Pitch.getNoteEnumString(transposedBassPitch.FundamentalNote);
             text += this.getTextForAccidental(transposedBassPitch.Accidental);
         }
+        console.log(text);
```

Comment:
remove console.log.
If anything, use `log.debug("OSMD: chord text: " + text)` or similar.
(don't just log a variable, also log what this variable is, and make this log searchable in the code (by adding "chord text: ..."))

Reply:
I need to figure out how to do that.

Reply:
Just write `log.debug(...)` as i wrote above, and add this to the imports at the top of the file:
`import log from "loglevel";`